lib/clubhouse_workflow.rb

```diff
@@ -104,6 +112,13 @@ def get_release_candidate_cards_titles(version_number)
 			}
 		end
 
+		private def get_cards_names_for_label_filteredby_feature(label)
```

Comment:
Since this is a script that the whole rider and driver vertical use, I would suggest making this method a little bit more dynamic by passing the filter as parameter. It wouldn't surprise me that tomorrow wanna get all cards filtered by "bug" only for example.

`private def get_cards_names(label, story_type)`

and then

`.select { |s| story_type ? contains_label(s,label) && s['story_type'] == story_type : contains_label(s,label) }`

Reply:
Ok, looks good. I am gonna add a new select though to avoid that hard to understand ternary